.gitattributes

```diff
@@ -5,28 +5,15 @@
 # to native line endings on checkout.
 
 # Common formats
-*.html text
```

Comment:
Not sure why modify this file. It's just a way to tell our VCS what the file types mean.

Reply:
I just want to avoid unnecessary code in all parts and eliminate unnecessary support questions.